.github/config.yml

```diff
@@ -254,6 +254,7 @@ run:
   mobile-core:
     paths:
     - "**/*"
```

Comment:
Did you mean to replace this line?

Reply:
no - it seems the glob matcher we are using doesnt recognize top level files with just this - i think we need both